### ISSUE TITLE
Tried pcsx on vita but gets error (C2-12828-1).

### ISSUE THREAD

**Issue by Fadi5555** [libretro/pcsx_rearmed#69]
As you see on the title I got error when I want to try play ps1 games.
Any solution please?

**Comment by frangarcj** [comment]
cue + bin format?

We need more data

**Comment by Fadi5555** [comment]
Yes yes I tried crash bandicoot 3 and it contains. Bin and cue. What should we do?
Can you support more format please.
Like (bin,iso,img,cue,sub,cdz,ccd, eboot.pbp) this all known ps1 format.

What is this core supporting currently?

Final when will dynarec version of iOS will available please.

Thanks a lot.
